partner_uom_mapping/views/partner_uom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 ACSONE SA/NV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record model="ir.ui.view" id="partner_uom_form_view">
+        <field name="name">partner.uom.form (in partner_uom_mapping)</field>
+        <field name="model">partner.uom</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <group>
+                        <field name="partner_id" />
+                        <field name="uom_id" />
+                        <field name="partner_uom" />
+                    </group>
+                </sheet>
+                <div class="oe_chatter" />
+            </form>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="partner_uom_search_view">
+        <field name="name">partner.uom.search (in partner_uom_mapping)</field>
+        <field name="model">partner.uom</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="partner_id" />
+                <field name="uom_id" />
+                <field name="partner_uom" />
+                <group expand="0" string="Group By">
+                    <filter
+                        string="Partner"
+                        name="group_by_partner"
+                        domain="[]"
+                        context="{'group_by': 'partner_id'}"
+                    />
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="partner_uom_tree_view">
+        <field name="name">partner.uom.tree (in partner_uom_mapping)</field>
+        <field name="model">partner.uom</field>
+        <field name="arch" type="xml">
+            <tree editable="top" decoration-muted="active == False">
+                <field name="partner_id" />
+                <field name="uom_id" />
+                <field name="partner_uom" />
+                <field name="active" invisible="1" />
+            </tree>
+        </field>
+    </record>
+
+    <record model="ir.actions.act_window" id="partner_uom_act_window">
+        <field name="name">Partner Units of Measure</field>
+        <field name="res_model">partner.uom</field>
+        <field name="view_mode">tree,form</field>
+        <field name="domain">[]</field>
+        <field name="context">{'search_default_group_by_partner': 1}</field>
+    </record>
+
+    <record model="ir.ui.menu" id="partner_uom_menu">
+        <field name="name">Partner Uom</field>
+        <field name="parent_id" ref="contacts.res_partner_menu_config" />
+        <field name="action" ref="partner_uom_act_window" />
+        <field name="groups_id" eval="[(4, ref('group_partner_uom'))]" />
```

Comment:
Isn't it a bit confusing that, giving the `group_partner_uom` let you access to the partner.uom in the UI, but cannot modify it if the user is not part of the `base.group_partner_manager` ?

I may be wrong, but, I was expecting that `group_partner_uom` allow `base.group_user` to see the menu and just read the `partner.uom`, and that `base.group_partner_manager` can always see this menu to read and write the `partner.uom`.

Reply:
README adapted.